hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -631,6 +631,13 @@ public static void assertIterableEquals(Iterable iterable, Object... values) {
         assertEquals(expected, actual);
     }
 
+    public static void assertSetEquals(Set actual, Object... values) {
```

Comment:
this is a bit unusual. Not really intuitive, to me anyway. Check `TestCollectionUtils`. You can use this:
```java
assertEquals(expected, setOf("1", "2", "3"));
```

This seems cleaner to me, it express its intend better.